### DISCUSSION TITLE
vc-viewer的sceneMode設為2D時,vc-drawings的editable失效?

### DISCUSSION THREAD

**Discussion by stan88** [zouyaoji/vue-cesium#341] [Q&A]
vc-viewer的sceneMode設為2D時,vc-drawings的editable設為TRUE
點上的編輯按鈕無法出現
但是回到示例中的原始地點(圖1)卻可以出現,
再繪制新圖形也能進行編輯,
再回到無法編輯的圖形地點還是無法編輯
像是[113.759407, 23.039070]
請問這是什麼問題?

圖1
![2023-02-09_090151](https://user-images.githubusercontent.com/5283322/217690155-3ca6ec7b-7bb6-4722-ab9d-472eec28e1c0.png)

**Comment by zouyaoji** [comment]
这个我测试没问题

**Reply by stan88** [reply]
剛再測試,一個圖形裡有某一個點不出現(圖裡紅框),
其他都可以
螢幕錄影
[Video_2023-02-09_111318.zip](https://github.com/zouyaoji/vue-cesium/files/10693200/Video_2023-02-09_111318.zip)
![2023-02-09_111509](https://user-images.githubusercontent.com/5283322/217708878-2fc424c3-2d1c-485b-8c89-8d6a0570e10f.png)

**Reply by stan88** [reply]
您再看這個錄影檔,按著拖拉後出現某個區域功能失效,
範圍的大小也依比例尺大小不同
螢幕錄影
[Video_2023-02-09_112626.zip](https://github.com/zouyaoji/vue-cesium/files/10693253/Video_2023-02-09_112626.zip)

**Comment by stan88** [comment]
您好,
這個問題是否能幫忙看看,
謝謝

**Reply by zouyaoji** [reply]
先mark一下。边线那个已经改了发布了，这个我再试试。

**Reply by zouyaoji** [reply]
:polygonDrawingOpts="polygonDrawingOpts"

```
const polygonDrawingOpts = {
   disableDepthTest: false
}
```

下个版本会将这个属性默认为 false。

**Reply by stan88** [reply]
您好,加了這個屬性,沒有改善
它好像有邊界似的,拉到LEVEL 3的層級
更明顯

**Reply by zouyaoji** [reply]
https://codepen.io/zouyaoji/pen/abaZWaJ

**Reply by stan88** [reply]
您好,
再看一下這視頻

[Video_2023-02-23_223356.zip](https://github.com/zouyaoji/vue-cesium/files/10814583/Video_2023-02-23_223356.zip)

**Reply by zouyaoji** [reply]
已重现，似乎只有三角形才有这个问题，我看看。

**Reply by zouyaoji** [reply]
https://github.com/zouyaoji/vue-cesium/blob/dev/packages/components/overlays/html/index.ts#L106

感谢反馈，已定位上述代码导致了问题，2D 模式下，VcOverlayHtml组件应该忽视autoHidden属性。

**Reply by zouyaoji** [reply]
已改，下个版本更新了就没有这个问题了。
![image](https://user-images.githubusercontent.com/24430116/220963231-ab3dcff8-0abd-461a-a5c6-d4de52c00d06.png)

**Reply by stan88** [reply]
您好，
太感謝了
